### PR TITLE
[BTFS-983] - added redis helpers and constant messages

### PR DESCRIPTION
**Pull request by uchetron**
Included redis function to create and check a redis connection

Included several constant messages to support functions that connect and check status of redis

### CODE REVIEW

File: constant/db.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ const (
 	DBMasterConnectionError = "Cannot connect to the master database!"
 	DBSlaveConnectionError  = "Cannot connect to the slave database!"
 	DBURLParseError         = "Cannot parse database connection URL!"
+	RDConnectionError 		= "Cannot connect to the RD database!"
+
```
Comment (wxue) [inline constant/db.go]: plz run ```go fmt ./...```

**Review by wxue — COMMENTED** [review]
need a approve from its user @JustinGaoF

File: db/redis/rd.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	rd *redis.Client
+}
+
+var (
+	host = ""
+	port = ""
+	db   = 0
+)
+
+func CreateTGRDDB(url string) *TGRDDB {
```
Comment (JustinGaoF) [inline db/redis/rd.go]: 1. Technically it is only connect to redis, not create db, using this function name will confuse other. 
2. We still need a function to just parse the three major variables for get connection. 
3. For the redis URI, there should combine many information, you can refer to https://github.com/lettuce-io/lettuce-core/wiki/Redis-URI-and-connection-details for it. while honestly I am not sure this URL is official right.

**Review by JustinGaoF — COMMENTED** [review]
Sorry for forgetting submit the comment before.

File: db/redis/rd.go (hunk 2 above)
Comment (uchetron) [reply db/redis/rd.go]: 1. ok 2. Will use parseRedisUrl function returning redis.Options, err 3. okay, i see.

**Review by JustinGaoF — APPROVED** [review]
LGTM

File: go.mod [hunk 3]
```diff
@@ -6,12 +6,9 @@ require (
...
 	github.com/go-pg/pg/v9 v9.0.1
+	github.com/go-redis/redis/v7 v7.0.0-beta.4
 	github.com/json-iterator/go v1.1.6
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
 	github.com/stretchr/testify v1.3.0
-	go.uber.org/atomic v1.4.0 // indirect
-	go.uber.org/multierr v1.1.0 // indirect
+	github.com/tron-us/go-common v1.0.2
```
Comment (wxue) [inline go.mod:12]: plz go mod tidy; plus make sure your changes based on go-common v2

Comment (uchetron) [reply go.mod:12]: okay..thnx

**Pull request merged**